meshes:
  - name: HV7_00
    base_frame: world
    mesh_pkg: bayes_pose
    path_from_pkg_to_mesh: model/meshes/STL
    mesh_name: HV7.stl
    x: 0.4
    y: 0.005
    z: 0.00
    qx: 0.0
    qy: 0.0
    qz: 0.0
    qw: 1.0
